libs/sdk-ui-charts/src/highcharts/chartTypes/_chartCreators/setupDrilldownToParentAttribute.ts

```diff
@@ -17,13 +17,14 @@ export function getDDPointsInParentTick(axis: any, tick: IHighchartsParentTick):
     }
 
     // replace y value by target value for bullet chart target
-    ddPoints.forEach((ddPoint) => {
+    const modifiedPoints = ddPoints.map((ddPoint) => {
         if ((ddPoint.series as any)?.userOptions?.bulletChartMeasureType === "target") {
-            ddPoint.y = ddPoint.isNullTarget ? null : ddPoint.target;
+            return Object.assign({}, ddPoint, { y: ddPoint.isNullTarget ? null : ddPoint.target });
```

Comment:
maybe I'm missing something, but what would you think about `return { ...ddPoint, y: ddPoint.isNullTarget ? null : ddPoint.target }`

Reply:
ddPoint is class instance with some methods and spread operator is not working on it